src/doctopus/storage/storage_impls/permanent_fs.clj

```diff
@@ -0,0 +1,24 @@
+(ns doctopus.storage.storage-impls.permanent-fs
+  (:require [doctopus
+             [configuration :refer [server-config]]
+             [files :as files]]
```

Comment:
My Clojure IDE refactored my imports this way and I can't decide how I feel about it. Thoughts?